flusher/flusher/handler.py

```diff
@@ -47,20 +48,38 @@ def handle_set_account(self, msg):
         )
 
     def handle_set_data_source(self, msg):
+        if msg["tx_hash"] is not None:
+            msg["transaction_id"] = self.conn.execute(
```

Comment:
Can we create a helper function to get transaction_id instead of copy these lines of code to be the same work?